templates/zerver/api/incoming-webhooks-walkthrough.md

```diff
@@ -202,6 +202,17 @@ After which you should see something similar to:
 2016-07-07 15:06:59,187 INFO     127.0.0.1       POST    200 143ms (mem: 6ms/13) (md: 43ms/1) (db: 20ms/9q) (+start: 147ms) /api/v1/external/helloworld (helloworld-bot@zulip.com via ZulipHelloWorldWebhook)
 ```
 
+Using the Integrations Developer Panel (GUI):
+After doing `./tools/run-dev.py` go to http://localhost:9991/devtools/integrations/.
+
+Select an *incoming webhook* bot, an integration and optionally a fixture which you can then further
+optionally tweak before sending. You may also optionally specify a stream and a topic.
```

Comment:
I would split this up into bullet-points. I would also recommend following the guidelines [here](https://zulip.readthedocs.io/en/latest/subsystems/integration-docs.html) when rewriting this section.

Reply:
Bullet points are a great idea, and I'll go through the guidelines.

Reply:
From what I've noticed, the current tutorial doesn't follow some of the guidelines, but I think that there's a good reason for this. Mainly, I'd like to reference this point:

> Cut unnecessary words. For example, do not start steps with transition words like “Next”, “First”, “Now”, etc. Each step should be structurally independent.

But these transition words are used heavily in the current tutorial. I believe that this is because this document is more of a tutorial and not a documentation. So should I strictly stick to the guidelines or just use my own judgement here?

Reply:
Hmm.. it is fine that it doesn't follow some of them. I just wanted to share those guidelines for future reference and for the limited scope of this particular section of the tutorial. Rishi helped me write those guidelines for our integrations docs and he's pretty amazing at this stuff! I would recommend following the guidelines for the bulleted section you add, but don't change the rest of the tutorial to follow those guidelines just yet. :)